scripts/clone-orchestra.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+REPO_DIR=orchestra
+
+if [ -d $REPO_DIR ]; then
+    rm -rf $REPO_DIR
+fi
+
+# Only supports master branch for duration of beta.
+REQUIRED_BRANCH="master"
+
+echo "Cloning branch $REQUIRED_BRANCH of orchestra..."
+git clone -b $REQUIRED_BRANCH https://github.com/StackStorm/orchestra.git ${REPO_DIR}
+
+echo "Copying files to local st2docs repo..."
+cp ${REPO_DIR}/docs/source/overview.rst ./docs/source/orchestra/overview.rst
```

Comment:
I would prefer to use a git subtree over this approach.

It's confusing and also causes build issues.